### PR TITLE
Update user's github attributes on login

### PR DESCRIPTION
**Pull request by cutwater**
Update `github_user` and `github_avatar` fields on user login.

Issue: #402

### CODE REVIEW

File: galaxy/accounts/models.py [hunk 1]
```diff
@@ -67,11 +69,14 @@ class CustomUser(AbstractBaseUser, PermissionsMixin, DirtyMixin):
     date_joined = models.DateTimeField(_('date joined'), default=timezone.now)
 
     karma = models.IntegerField(default=0, db_index=True)
-    github_avatar = models.CharField(_('github avatar'), max_length=254, blank=True)
-    github_user = models.CharField(_('github user'), max_length=254, blank=True)
-    cache_refreshed = models.BooleanField(_('cache refreshed'), default=False)
-
-    objects = UserManager()
+    github_avatar = models.CharField(
```
Comment (chouseknecht) [inline galaxy/accounts/models.py]: I'm confused. I thought we were changing `github_user` and `github_avatar` to be properties, with the results pulled from the social account table.

Comment (cutwater) [reply galaxy/accounts/models.py]: According to discussion in slack, let's rename it to `avatar` as a part of this PR.

File: galaxy/main/signals/handlers.py [hunk 2]
```diff
@@ -37,21 +36,29 @@
 
 @receiver(user_logged_in)
 def user_logged_in_handler(request, user, **kwargs):
+    social = auth_models.SocialAccount.objects.get(
+        provider=constants.PROVIDER_GITHUB.lower(), user=user)
+    user.avatar_url = social.extra_data.get('avatar_url')
```
Comment (chouseknecht) [inline galaxy/main/signals/handlers.py]: This is fine for now. However, in the future, I anticipate we'll have a page where the user can manage their profile, and update their `avatar_url`. When that becomes true, we should only set the `user.avatar_url` when it is empty.

Comment (cutwater) [reply galaxy/main/signals/handlers.py]: I'm totally agree.

**Pull request merged**